### ISSUE TITLE
Standalone plot window on Linux

### ISSUE THREAD

**Issue by lnacquaroli** [jheinen/GR.jl#96]
I cannot get a standalone window with GR in Atom. The following will open a window in atom pane only:
`using GR`
`plot(rand(20))`

With Plots actually does not show anything at all, even in the atom plot pane:
`using Plots`
`plt = plot(rand(20))`
`gui()`

If I execute only the line `plot(rand(19))` it will plot in the atom pane. Or if I include the line display(plt), it will plot in the atom pane also but not a standalone window.

I might be missing something. Notice that in the REPL works fine though. 

Thanks.

`Pkg.status("GR")`
` - GR                            0.25.0+            master`

**Comment by jheinen** [comment]
Could you please update GR. The most recent version is 0.28.0 (runtime 0.30.0)

**Comment by lnacquaroli** [comment]
Great, I see it now, little buggy though. Thanks!

**Comment by jheinen** [comment]
With the new GKS Qt terminal, you might get better (resizable) output. Or, what do you mean with 'buggy' ?
```
ENV["GKSwstype"]="gksqt"
using Plots
plot(randn(20))
```
<img width="639" alt="screen shot 2018-03-08 at 19 56 18" src="https://user-images.githubusercontent.com/5718244/37170329-de1d33aa-230a-11e8-9c5b-dbda00b20d9d.png">

**Comment by lnacquaroli** [comment]
I see now the problem. It was buggy because I wasn't setting the `gksqt` value, so it was using the old terminal.

I tried your code in the REPL and it worked. Note that If I run
`ENV["GKSwstype"] = "gksqt"`
`using Plots`
`plt=plot(randn(20))`
`gui(plt)`

from the atom editor, it doesn't pop-up any plot in a window nor in the plot pane. If I use

`ENV["GKSwstype"] = "gksqt"`
`using Plots`
`plt=plot(randn(20))`
`display(plt)`

it shows the plot in the atom plot pane, but not the standalone window. Now, if I do

`ENV["GKSwstype"] = "gksqt"`
`using Plots`
`plt=plot(randn(20))`
`display(plt)`
`gui(plt)`

it shows several plots (from the current and previous executions of the script) plus the one in the atom pane.

**Comment by jheinen** [comment]
The GKS Qt terminal is not yet the default output device for the GR backend in Plots. This means, what you observed is currently the expected behaviour.

**Comment by lnacquaroli** [comment]
I thought so, but I insisted because it does work well apparently with Plots in the REPL though.

Thank you!

**Comment by jheinen** [comment]
The multiple window support is on the way ...

**Comment by lnacquaroli** [comment]
When running `Plots` inside Atom editor I got the old terminal that pops up instead of the "gksqt" after updating to GR 0.31. I tried `ENV["GKSwstype"] = "gksqt"`, and `ENV["GRDIR"]=""; Pkg.build("GR")` but nothing changes. Running in REPL the new terminal pops up though well.

**Comment by jheinen** [comment]
Do you expect the plot to appear in a separate window when using Atom, nteract, Julia etc?

**Comment by lnacquaroli** [comment]
Yeah, so I was reading that the new version of GR would be supporting multi-windows, but not sure if it's working so far. 

To answer your question, yes. I am using Atom with julia and I would like to have possibility to open multiple windows with the gui() command from Plots (just like I am using now the pyplot() backend). Again, not sure how far this is implemented on GR, so I'm just wondering.

Thank you.

**Comment by jheinen** [comment]
Sorry, it's not yet implemented. The multi-window support is still WIP - we have to make major changes to reflect this both in console and GUI environments.
